third_party/yara/JPCERT/APT10/apt10.yara

```diff
@@ -79,7 +79,7 @@ rule APT10_Himawari_strings {
         author = "JPCERT/CC Incident Response Group"
         rule_usage = "memory scan"
         reference = "https://www.jpcert.or.jp/present/2018/JSAC2018_01_nakatsuru.pdf"
-        hash1 = "3938436ab73dcd10c495354546265d5498013a6d17d9c4f842507be26ea8fafb"
```

Comment:
Should we ignore anything in `third_party` since any refreshes/updates will overwrite the changes?

Reply:
yeah, that was inadvertent - will fix.